Sources/PointFree/Episode/Show.swift

```diff
@@ -7,33 +7,36 @@ import HtmlCssSupport
 import HttpPipeline
 import HttpPipelineHtmlSupport
 import Models
+import PointFreePrelude
 import PointFreeRouter
 import Prelude
 import Styleguide
 import Tuple
 import Views
 
-let episodeResponse =
+let episodeResponse: M<Tuple5<Either<String, Episode.Id>, User?, SubscriberState, Route?, Episode.Collection.Slug?>> =
```

Comment:
middleware now takes an optional collection slug for the case that you are viewing this episode in the context of a collection.